ansible/scripts/get-endpoints.sh

```diff
@@ -1,4 +1,4 @@
-NUM_CONNECTIONS=1
+NUM_CONNECTIONS=4 #TODO: pass this value from Makefile
```

Comment:
```suggestion
NUM_CONNECTIONS=${1-4} #TODO: pass this value from Makefile
```

Reply:
This takes the first parameter passed to the script, if set, otherwise the default = 4.